### PR TITLE
Prepare for cansi v3 release

### PR DESCRIPTION
**Pull request by barafael**
Some `cansi` functions will be marked as deprecated with `cansi` v3.

See here: https://docs.rs/cansi/latest/cansi/v3/index.html

Let me know what you think :)

### CODE REVIEW

File: src/output.rs [hunk 1]
```diff
@@ -212,28 +212,32 @@ fn format_output(ui: &mut Ui, text: &str) {
...
                         }
 
-                        text = match intensity {
-                            cansi::Intensity::Normal => text,
-                            cansi::Intensity::Bold => text.strong(),
-                            cansi::Intensity::Faint => text.weak(),
+                        text = match intensity.unwrap_or(Intensity::Normal) {
+                            Intensity::Normal => text,
+                            Intensity::Bold => text.strong(),
+                            Intensity::Faint => text.weak(),
```
Comment (MichalGniadek) [inline src/output.rs:240]: Minor change, but I think I slightly prefer this version:
```
text = match intensity {
    Some(Intensity::Bold) => text.strong(),
    Some(Intensity::Faint) => text.weak(),
    Some(Intensity::Normal) | None => text,
};
```

**Review by MichalGniadek — CHANGES_REQUESTED** [review]
Thank you, I just have a minor nitpick :)

File: src/output.rs (hunk 1 above)
Comment (barafael) [reply src/output.rs:239]: That's better. Done.

**Review by MichalGniadek — CHANGES_REQUESTED** [review]
I think you forgot the `Some(...)`

**Comment by barafael** [comment]
That's right! Fixed (this time, from a computer not the GH web page)

**Review by MichalGniadek — APPROVED** [review]

**Comment by MichalGniadek** [comment]
Thank you!

**Pull request merged**